app/pb_kits/playbook/pb_image/_image.jsx

```diff
@@ -14,7 +14,7 @@ const Image = ({
   <img
       alt={alt}
       className="pb_image lazyload blur_up"
-      data-src={url}
+      src={url}
```

Comment:
We probably want to revert this since the `data` attribute is here for use with our `lazy-sizes` image loader library.